Sources/KlaviyoSwift/InternalAPIModels.swift

```diff
@@ -190,7 +209,24 @@ extension Klaviyo.Profile.Attributes.Location: Encodable {
  Encoding
  */
 
-extension Klaviyo.Event.Attributes.Metric: Encodable {
+extension Klaviyo.Event: Encodable {
+    enum CodingKeys: CodingKey {
+        case attributes
+    }
+    
+    public func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: CodingKeys.self)
+        try container.encode(attributes, forKey: .attributes)
+    }
+}
+
+extension Klaviyo.Event.Attributes.Metric: Codable {
+    init(from decoder: Decoder) throws {
+        let values = try decoder.container(keyedBy: CodingKeys.self)
+//        self.service = try values.decode(String.self, forKey: .service)
```

Comment:
accident?

Reply:
No actually it's a constant for now...I should probably just remove that or change how it's set.